source/text/Format.cpp

```diff
@@ -293,20 +293,23 @@ string Format::Credits(int64_t value)
 	int64_t absolute = abs(value);
 
 	// Handle numbers bigger than a million.
-	static const vector<char> SUFFIX = {'T', 'B', 'M'};
-	static const vector<int64_t> THRESHOLD = {1000000000000ll, 1000000000ll, 1000000ll};
-	for(size_t i = 0; i < SUFFIX.size(); ++i)
-		if(absolute > THRESHOLD[i])
+	static constexpr array<pair<int64_t, char>, 3> THRESHOLD_SUFFIX = {{
+		{1000000000000ll, 'T'},
```

Comment:
Should we use separators for large numbers, for readability?
E.g. 
```cpp
1'000'000'000'000ll
```

Reply:
I'd vote yes, but it would be reasonable to add them everywhere, in a separate PR

Reply:
Sounds reasonable. Let's keep that for a separate PR (since it likely triggers quite a bit of disucssion).